irods/test/admin_test.py

```diff
@@ -73,6 +73,29 @@ def test_create_delete_user_zone(self):
             self.sess.users.get(self.new_user_name, self.sess.zone)
 
 
+    def test_groupadmin_creates_user_group__374(self):
+        # Have user with groupadmin
+        user = self.sess.users.create(self.new_user_name, 'groupadmin')
+        self.assertEqual(user.name, self.new_user_name)
+
+        # Have a password for login
+        user_password = 'bpass'
+        user.modify('password', user_password)
+
+        user_group_name = 'coolgroup'
+
+        # Create session as user
+        with iRODSSession(port=self.sess.port, zone=self.sess.zone, host=self.sess.host,
+                          user=self.new_user_name, password=user_password) as bobby:
+            # Create group
+            group = bobby.user_groups.create(user_group_name)
+            self.assertEqual(group.name, user_group_name)
+
+        # Only an admin can remove user_groups!
```

Comment:
ah, did you determine this to be the case?    you tried as the groupadmin, and it did not work?   if so, please add an assertion with that as well - so we will remember next time.

Reply:
When I tried removing the group with the groupadmin, it wouldn't allow it. This matches up with the iRODS docs for igroupadmin too.

I'll go ahead and add an assertion that groupadmin raises an exception when trying to remove the group.

Reply:
Nice - points for the docs!